.envs/.local/.sls

```diff
@@ -0,0 +1,7 @@
+MINIO_ACCESS_KEY=enrolments
+MINIO_SECRET_KEY=wJalrXUtnFEMIK7MDENGbPxRfiCYEXAMPLEKEY
+S3_ACCESS_KEY_ID=enrolments
+S3_SECRET_ACCESS_KEY=wJalrXUtnFEMIK7MDENGbPxRfiCYEXAMPLEKEY
+S3_ENDPOINT_URL=http://minio:9000
+CALLBACK_BUCKET=put-callbacks-here
+ENROLMENT_BUCKET=put-enrolments-here
```

Comment:
I think we don't need them anymore. Did you check this PR: https://github.com/ACWIC/employer-admin/pull/27